src/lib/utils/invokeApiRequest.ts

```diff
@@ -9,6 +9,7 @@ import {
   type HttpVerb,
   ResponseType,
 } from "@tauri-apps/api/http";
+import { addInfoLog } from "$lib/stores/debugLogStore";
```

Comment:
Let's revert this, as it's just an unused import

Reply:
Agreed, this was just for debugging